tests/test_traced_data.py

```diff
@@ -0,0 +1,301 @@
+import collections
+import time
+import unittest
+import six
+
+from core_data_modules.traced_data import TracedData, Metadata
+
+
+class TestTracedData(unittest.TestCase):
+    @staticmethod
+    def td_1():
+        data = {"id": "0", "phone": "+441632000001", "gender": "man"}
+        return TracedData(data, Metadata("test_user", "run_fetcher", time.time()))
+
+    @classmethod
+    def td_2(cls, td):
```

Comment:
td_2 has a very different signature to td, which implies that its somewhat different in purpose (I think it's appending data to an existing traced data rather than creating one?) -- in which case it would be better for it to have a more different name.